attributes/default.rb

```diff
@@ -395,9 +395,12 @@
 )
 # Lustre repo string is built following the official doc
 # https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html
+# 'centos' is used for arm and 'el' for x86_64
+fsx_arm_prefix = arm_instance? ? 'centos' : 'el'
 default['cfncluster']['lustre']['base_url'] = value_for_platform(
   'centos' => {
     # node['kernel']['machine'] contains the architecture: 'x86_64' or 'aarch64'
+    '>=7' => "https://fsx-lustre-client-repo.s3.amazonaws.com/#{fsx_arm_prefix}/7/#{node['kernel']['machine']}",
```

Comment:
I think we need to do the following:
* Keep `el/7.#{find_rhel_minor_version}` logic for x86 or verify with FSx team to see if we still need this logic. This logic was added in [this commit](https://github.com/aws/aws-parallelcluster-cookbook/commit/2e4932cd5fdddb60fa68ffb3de39027f300823cc) and seems necessary to support different minor versions.
* If urls for x86 and arm are very different, we might want do a conditional in `value_for_platform`, something like `'>=7' => arm_instance? ? <arm_url> : <x86_url>`
* As is, this url will be used for x86 and ARM CentOS >=7.7 anyway and we can keep the existing code by using `'default' =>` instead of `'>=7' =>`

Reply:
Lots to unpack here:

1. I removed the `>=7` option and included that logic in `default`
2. I added / kept the minor version pinning. I think this is important.
3. No need for the conditional url. 
4. Yup done